crates/db_schema/src/lib.rs

```diff
@@ -277,12 +277,27 @@ pub enum CommunityVisibility {
   /// Public community, any local or federated user can interact.
   #[default]
   Public,
-  /// Unfederated community, only local users can interact.
-  LocalOnly,
+  /// Posts from this community do not appear on Local and All feeds.
+  Hidden,
+  /// Unfederated community, only local users can interact (with or without login).
+  LocalOnlyPublic,
+  /// Unfederated  community, only logged-in local users can interact.
+  LocalOnlyPrivate,
   /// Users need to be approved by mods before they are able to browse or post.
   Private,
 }
```

Comment:
Let me know if you have better suggestions how these should be called.

Reply:
To make this change less confusing, rename Hidden to Unlisted.